Assets/Scripts/Model/Content/SecondEdition/Pilots/Rebel/RZ1AWing/TychoCelchuBoE.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Abilities.SecondEdition;
+using Actions;
+using ActionsList;
+using Content;
+using Ship;
+using System.Collections.Generic;
+using Tokens;
+using Upgrade;
+using UpgradesList.SecondEdition;
+
+namespace Ship
+{
+    namespace SecondEdition.RZ1AWing
+    {
+        public class TychoCelchuBoELSL : RZ1AWing
+        {
+            public TychoCelchuBoELSL() : base()
+            {
+                PilotInfo = new PilotCardInfo25(
+                    "Tycho Celchu",
+                    "Battle Over Endor",
+                    Faction.Rebel,
+                    5,
+                    4,
+                    loadoutValue: 0,
+                    isLimited: true,
+                    abilityType: typeof(TychoCelchuBattleOverEndorAbility),
+                    tags: new List<Tags>
+                    {
+                        Tags.AWing
+                    },
+                    extraUpgradeIcons: new List<UpgradeType> {
+                        UpgradeType.Talent, 
+                        UpgradeType.Talent,
+                        UpgradeType.Missile,
+                        UpgradeType.Modification,
+                        UpgradeType.Configuration
+                    },
+                    isStandardLayout: true
+                );
+
+                ImageUrl = "https://infinitearenas.com/xw2/images/quickbuilds/tychocelchu-battleoverendor.png";
+
+                PilotNameCanonical = "tychocelchu-battleoverendor-lsl";
+
+                ShipInfo.Shields++;
+
+                ShipInfo.ActionIcons.AddLinkedAction(new LinkedActionInfo(typeof(FocusAction), typeof(ReloadAction)));
+                ShipInfo.ActionIcons.AddLinkedAction(new LinkedActionInfo(typeof(BoostAction), typeof(EvadeAction)));
+
+                MustHaveUpgrades.Add(typeof(VectoredCannonsRZ1));
+                MustHaveUpgrades.Add(typeof(ItsATrap));
+                MustHaveUpgrades.Add(typeof(Juke));
+                MustHaveUpgrades.Add(typeof(ProtonRockets));
+                MustHaveUpgrades.Add(typeof(ChaffParticlesBoE));
+            }
+        }
+    }
+}
+
+namespace Abilities.SecondEdition
+{
+    public class TychoCelchuBattleOverEndorAbility : GenericAbility
+    {
+        //While you are disarmed, you can still perform missile attacks. When you perform a missile attack while disarmed, roll a maximum of 4 dice.
+        GenericSpecialWeapon secondaryWeapon;
+
+        public override void ActivateAbility()
+        {
+            HostShip.OnWeaponsDisabledCheck += AllowMissileAttacks;
+        }
+
+        public override void DeactivateAbility()
+        {
+            HostShip.OnWeaponsDisabledCheck -= AllowMissileAttacks;        
+        }
+
+        private void AllowMissileAttacks(ref bool result)
```

Comment:
Edge case but would this override the minimum range on Proton Rockets? I was trying it out and I'm not sure if Darth Vader actually bumped into Tycho or it was just really really close, but it did let me use Prockets to attack him.